src/ScrumBoardItBundle/Controller/DefaultController.php

```diff
@@ -129,8 +129,7 @@ public function addFlagAction(Request $request)
             $apiService = $this->get($this->getUser()->getApi());
             $selected = $request->request->get('issues');
             $apiService->addFlag($request, $selected);
-        } catch(\Exception $e) {
-            
+        } catch (\Exception $e) {
```

Comment:
Remove this or log exception, it stifles all exceptions. It's best way to lose time in futur to find why you don't see exception